hugolib/site.go

```diff
@@ -99,7 +99,7 @@ type SiteInfo struct {
 	Social                SiteSocial
 	Sections              Taxonomy
 	Pages                 *Pages
-	Files                 []*source.File
+	Files                 *[]*source.File
```

Comment:
nope, we were wrong - turns out i needed to rebase, and then to delete some debug ERROR logging :(